Bottom: 67cf298c63fd5a648133a33eda2fa7dae45b27e4
Top:    7b5a5da32ab6fe32f2fa6ec08692b7d853fefef9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:28:57 +0000

Refresh of make-poly_copy1-a-static

---

diff --git a/src/polyarea.h b/src/polyarea.h
index be95bd2..fc6a63d 100644
--- a/src/polyarea.h
+++ b/src/polyarea.h
@@ -144,7 +144,6 @@ BOOLp poly_M_Copy0(POLYAREA ** dst, const POLYAREA * srcfst);
 void poly_M_Incl(POLYAREA **list, POLYAREA *a);
 
 BOOLp poly_Copy0(POLYAREA **dst, const POLYAREA *src);
-BOOLp poly_Copy1(POLYAREA  *dst, const POLYAREA *src);
 
 BOOLp poly_InclContour(POLYAREA * p, PLINE * c);
 BOOLp poly_ExclContour(POLYAREA * p, PLINE * c);
diff --git a/src/polygon1.c b/src/polygon1.c
index b5ef082..663dd11 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2968,20 +2968,7 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
 /**********************************************************************/
 /* polygon routines */
 
-BOOLp
-poly_Copy0 (POLYAREA ** dst, const POLYAREA * src)
-{
-  *dst = NULL;
-  if (src != NULL)
-    *dst = (POLYAREA *)calloc (1, sizeof (POLYAREA));
-  if (*dst == NULL)
-    return FALSE;
-  (*dst)->contour_tree = r_create_tree (NULL, 0, 0);
-
-  return poly_Copy1 (*dst, src);
-}
-
-BOOLp
+static BOOLp
 poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
 {
   PLINE *cur, **last = &dst->contours;
@@ -2999,6 +2986,19 @@ poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
   return TRUE;
 }
 
+BOOLp
+poly_Copy0 (POLYAREA ** dst, const POLYAREA * src)
+{
+  *dst = NULL;
+  if (src != NULL)
+    *dst = (POLYAREA *)calloc (1, sizeof (POLYAREA));
+  if (*dst == NULL)
+    return FALSE;
+  (*dst)->contour_tree = r_create_tree (NULL, 0, 0);
+
+  return poly_Copy1 (*dst, src);
+}
+
 void
 poly_M_Incl (POLYAREA ** list, POLYAREA * a)
 {
